aicssegmentation/workflow/workflow_engine.py

```diff
@@ -4,6 +4,8 @@
 from .workflow import Workflow
 from .workflow_definition import WorkflowDefinition
 from .structure_wrapper_config import StructureWrapperConfig
+from os import path
```

Comment:
Use `Pathlib.Path` instead of `os.path`, it's the new de-facto Python path library.